action.yml

```diff
@@ -25,6 +17,34 @@ inputs:
     description: "Additional arguments for stack invocations"
```

Comment:
Let's expand this description a bit now that this applies to all `stack` commands and is global arguments. Not sure exactly how to phrase it though.

Reply:
tried to fix this in https://github.com/freckle/stack-action/pull/35/commits/a24c11aa06942907da24e0e0ab2ab81f1ec259b6